database/migration/src/main/resources/metadata_changesets/add_metadata_archive_status.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
This xml tag doesn't specify an `objectQuotingStrategy` attribute, which must be set for PostgreSQL to be uppercase like we do for all other DBMS.

https://github.com/broadinstitute/cromwell/blob/5ad2879795f6c6017ff5e4af440d55a232182664/database/migration/src/main/resources/metadata_changesets/postgresql_metadata_schema.xml#L2-L5